app/plugin.go

```diff
@@ -370,7 +374,17 @@ func (a *App) GetActivePluginManifests() ([]*model.Manifest, *model.AppError) {
 // activation if inactive anywhere in the cluster.
 // Notifies cluster peers through config change.
 func (a *App) EnablePlugin(id string) *model.AppError {
-	return a.ch.enablePlugin(id)
+	appErr := a.checkIfIntegrationMeetsFreemiumLimits(id)
```

Comment:
Should app/onboarding.go also perform this check, and/or should it be updated to use this public EnabledPlugin method? I'm imagining it would be especially relevant if/when the onboarding screen brings up a user-selectable menu of many plugins; though possibly even now a tricky user could create an abusive payload with too many plugins to be enabled for onboarding.

Reply:
@neallred I think calling the public method makes the most sense. This scenario means the admin went out of their way to sabotage the process, so it's appropriate to fail. Ideally it shouldn't affect other parts of onboarding.